test/screenshot/golden.json

```diff
@@ -44,4 +44,4 @@
   "drawer/modal": "da83487c9349b253f7d4de01f92d442de55aab92a8028b77ff1a48cfaa265b72",
   "drawer/permanentToModal": "6355905c2241b5e6fdddc2e25119a1cc3b062577375a88b59e6750c4b76e4561",
   "typography": "c5e87d672d8c05ca3b61c0df4971eabe3c6a6a1f24a9b98f71f55a23360c498a"
-}
+}
```

Comment:
add new line...

Also once we're finalized. I will create the new screenshots and I'll have you update here.